2.1.0 package references non existant Xamarin pacakage and is uninstallable

The 2.1.0 release is not installable via the nuget package manager.  It appears there is a typo in the nuspec file.  When trying to install this error occurs:

```
1>C:\Program Files\dotnet\sdk\7.0.401\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.Shared.targets(152,5): warning NETSDK1023: A PackageReference for 'Xamarin.Google.Crypto.Tink.Android' was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs
1>E:\Zecil\Board Game Copilot\frontend-blazor\Board Game Copilot\Board Game Copilot.csproj : error NU1605: Warning As Error: Detected package downgrade: Xamarin.Google.Crypto.Tink.Android from 2.1.0 to 1.8.0.1. Reference the package directly from the project to select a different version. 
1>E:\Zecil\Board Game Copilot\frontend-blazor\Board Game Copilot\Board Game Copilot.csproj : error NU1605:  Board Game Copilot -> Kebechet.Maui.RevenueCat.InAppBilling 2.1.0 -> Xamarin.Google.Crypto.Tink.Android (>= 2.1.0) 
1>E:\Zecil\Board Game Copilot\frontend-blazor\Board Game Copilot\Board Game Copilot.csproj : error NU1605:  Board Game Copilot -> Xamarin.Google.Crypto.Tink.Android (>= 1.8.0.1)
1>Done building project "Board Game Copilot.csproj" -- FAILED.
```

I suspect this may have been an attempt to fix #13 but the nuspec file references v 2.1.0 of this package which does not exist, and seems like a typo since its the same version as the current release.  The nuspec looks like this:

```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
  <metadata>
    <id>Kebechet.Maui.RevenueCat.InAppBilling</id>
    <version>2.1.0</version>
    <authors>Maui.RevenueCat.InAppBilling</authors>
    <license type="file">LICENSE</license>
    <licenseUrl>https://aka.ms/deprecateLicenseUrl</licenseUrl>
    <readme>README.md</readme>
    <projectUrl>https://github.com/Kebechet/Maui.RevenueCat.InAppBilling</projectUrl>
    <description>Serves as an abstraction around RevenueCat Android and iOS SDKs https://docs.revenuecat.com/docs</description>
    <releaseNotes>Added Entitlements into CustomerInfoDto</releaseNotes>
    <repository type="git" url="https://github.com/Kebechet/Maui.RevenueCat.InAppBilling" />
    <dependencies>
      <group targetFramework="net7.0-android33.0">
        <dependency id="Tonestro.Maui.RevenueCat.Android" version="6.5.0.1" exclude="Build,Analyzers" />
        <dependency id="Xamarin.Google.Crypto.Tink.Android" version="2.1.0" exclude="Build,Analyzers" />
      </group>
      <group targetFramework="net7.0-ios16.1">
        <dependency id="Kebechet.Maui.RevenueCat.iOS" version="4.19.0" exclude="Build,Analyzers" />
      </group>
      <group targetFramework="net7.0-maccatalyst16.1" />
      <group targetFramework="net7.0-windows10.0.19041" />
    </dependencies>
  </metadata>
</package>
```

The most recent version of Xamarin.Google.Crypto.Tink.Android is [1.10.0.1](https://www.nuget.org/packages/Xamarin.Google.Crypto.Tink.Android/1.10.0.1) so this can't possibly be right.

--- a/Maui.RevenueCat.InAppBilling/Maui.RevenueCat.InAppBilling.csproj
+++ b/Maui.RevenueCat.InAppBilling/Maui.RevenueCat.InAppBilling.csproj
@@ -29,8 +29,8 @@
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <DebugSymbols>true</DebugSymbols>
     <DebugType>embedded</DebugType>
-    <Version>2.1.0</Version>
-    <PackageReleaseNotes>Added Entitlements into CustomerInfoDto</PackageReleaseNotes>
+    <Version>2.1.1</Version>
+    <PackageReleaseNotes>Fixed package versions</PackageReleaseNotes>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net7.0-ios'">
@@ -40,7 +40,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net7.0-android'">
     <PackageReference Include="Tonestro.Maui.RevenueCat.Android" Version="6.5.0.1" />
-    <PackageReference Include="Xamarin.Google.Crypto.Tink.Android" VersionOverride="1.8.0.1" />
+    <PackageReference Include="Xamarin.Google.Crypto.Tink.Android" Version="[1.8.0.1,)" />
   </ItemGroup>
 
   <ItemGroup>

